src/components/wagonwheelComponent/chartPane/index.js

```diff
@@ -86,7 +93,7 @@ class chartPane extends Component {
     drawDots(x,y) {
         const canvas = this.refs.canvas
         let ctx = canvas.getContext("2d");
-        let pointSize=2
+        let pointSize = 2
         ctx.fillStyle = "#6b6b6b"; // Red color
```

Comment:
Give meaningful names. Instead of fillStyle you may call it fillColor.

Reply:
I think its the property of canvas/ . fillStyle